### PR TITLE
Add FAQ section in navDrawer

### LINKED ISSUE #73: feat: Add faq section in navigation drawer
**Summary:**

Add the faq section in the application

**Steps to reproduce:**

*  Open Application
*  Open Navigation drawer
*  No faq section

**Expected behaviour:**

 This task is to add faq functionality.

### PR DESCRIPTION
**Pull request by ashwinkey04**
Fixes #73 

**Screenshot** (Device: Asus Max Pro M2 , ROM: OxygenOS 10)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by rachittt** [comment]
hey @ashwinkey04, we are currently working on a complete redesign of Mifos Mobile CN. The redesign does not have a navigation drawer.

**Comment by ashwinkey04** [comment]
Hey @rachittt Thanks for your inputs. Will the options in the navDrawer be moved to some other menu in, say settings?

**Comment by rachittt** [comment]
> Hey @rachittt Thanks for your inputs. Will the options in the navDrawer be moved to some other menu in, say settings?
Yes, it will be there on the top left corner of the profile menu. Check the screenshot in #112

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt [hunk 1]
```diff
@@ -221,5 +224,8 @@ class DashboardActivity : MifosBaseActivity(), View.OnClickListener, NavigationV
         val intent = Intent(this, AboutUsActivity::class.java)
         startActivity(intent)
     }
+    fun showFaq(){
+        Toast.makeText(applicationContext,"FAQ section yet to be added",Toast.LENGTH_SHORT).show()
```
Comment (laxyapahuja) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt]: hey. kindly use the ```Toaster``` class in utils :) check line 215 for reference.

**Comment by rachittt** [comment]
@ashwinkey04 Squash the commits

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt (hunk 1 above)
Comment (ashwinkey04) [reply app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt]: Done ;)

**Comment by garvit984** [comment]
@ashwinkey04 The app has been redesigned so this PR is now not relatable.

**Pull request closed without merging**